### DISCUSSION TITLE
Add to conda-forge recipes?

### DISCUSSION THREAD

**Discussion by dwsmith1983** [zifeo/dataconf#53] [Ideas]
@zifeo should the library be added to conda-forge to make it easy on Anaconda users?

https://github.com/conda-forge/staged-recipes

**Comment by zifeo** [comment]
@dwsmith1983 Good idea! It seems it was already done, see https://anaconda.org/conda-forge/dataconf and https://github.com/conda-forge/staged-recipes/pull/16841. However might be worth to see how to support official this channel?

**Reply by dwsmith1983** [reply]
Okay, yeah it is available. Surprised someone did it.

https://anaconda.org/conda-forge/dataconf

**Reply by dwsmith1983** [reply]
Any ideas on how to support or interact with the conda group?

**Reply by zifeo** [reply]
@dwsmith1983 I believe you may want to reach out to [thewchan](https://github.com/thewchan) and ask him to [update](https://conda-forge.org/docs/maintainer/updating_pkgs.html#updating-the-maintainer-list) the maintainer list.

**Reply by dwsmith1983** [reply]
I just followed the instructions on and submitted it to conda-forge admins
https://github.com/conda-forge/dataconf-feedstock
